src/test/cloudWatchLogs/explorer/logGropNode.test.ts

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import { CloudWatchLogs } from 'aws-sdk'
+import * as os from 'os'
+import { LogGroupNode } from '../../../cloudWatchLogs/explorer/logGroupNode'
+import { ext } from '../../../shared/extensionGlobals'
+import { TestAWSTreeNode } from '../../shared/treeview/nodes/testAWSTreeNode'
+import { clearTestIconPaths, IconPath, setupTestIconPaths } from '../../shared/utilities/iconPathUtils'
+
+describe('LogGroupNode', () => {
+    const parentNode = new TestAWSTreeNode('test node')
+    let testNode: LogGroupNode
+    let fakeLogGroup: CloudWatchLogs.LogGroup
+
+    before(async () => {
+        setupTestIconPaths()
+        fakeLogGroup = {
+            logGroupName: "it'sBig/it'sHeavy/it'sWood",
+            arn: "it'sBetterThanBadIt'sGood",
+        }
+
+        testNode = new LogGroupNode(parentNode, 'someregioncode', fakeLogGroup)
+    })
+
+    after(async () => {
+        clearTestIconPaths()
+    })
+
+    it('instantiates without issue', async () => {
+        assert.ok(testNode)
+    })
+
+    it('initializes the parent node', async () => {
+        assert.strictEqual(testNode.parent, parentNode, 'unexpected parent node')
+    })
+
+    it('initializes the region code', async () => {
+        assert.strictEqual(testNode.regionCode, 'someregioncode')
+    })
+
+    it('initializes the label', async () => {
+        assert.strictEqual(testNode.label, fakeLogGroup.logGroupName)
+    })
+
+    it('initializes the functionName', async () => {
+        assert.strictEqual(testNode.logGroupName, fakeLogGroup.logGroupName)
+    })
+
+    it('initializes the tooltip', async () => {
+        assert.strictEqual(testNode.tooltip, `${fakeLogGroup.logGroupName}${os.EOL}${fakeLogGroup.arn}`)
```

Comment:
concept of "the widget initializes itself" is trivial, so I would put all of these tests in 1 test.

```
it('initializes properties'), async() => {
  assert.strictEqual(testNode.regionCode, 'someregioncode')
  assert.strictEqual(testNode.label, fakeLogGroup.logGroupName)
  ...
})
```

Reply:
This is a copy/paste of some of the other resource node tests. I'll leave it out of this so we can do a full pass on these later (I'd also like to explore cutting down on the repeated code, I'll leave that for another day).